src/Widgets/JobRow.vala

```diff
@@ -21,15 +21,19 @@
  */
 
 public class Printers.JobRow : Gtk.ListBoxRow {
-    public Job job { get; private set; }
-    private Printer printer;
+    public Job job { get; construct set; }
+    public Printer printer { get; construct; }
```

Comment:
use `construct set;` like other vars

Reply:
I would instead propose the remove the set of the job one

Reply:
Job needs a set because it's changed in a method below

Reply:
so `private set` would be good for me